index.html

```diff
@@ -176,7 +207,15 @@ <h3 id="form-heading">
         </h3>
         <form action="https://formspree.io/f/xyyvbjao" method="post">
```

Comment:
Please make sure to stick with the design specifications provided on the Figma template (e.g., font, colors, images, text, margins). Kindly adjust the font size and width of the paragraph beside your contact form to align with your chosen Figma design template.

> Your design
![newcap contact](https://user-images.githubusercontent.com/71140133/200094560-b72223ea-b53f-4ed6-b176-5a5bfcf9882a.PNG)

> Figma design template
![newcap](https://user-images.githubusercontent.com/71140133/200094576-6f239404-ab06-4100-8ba9-be32aed68579.PNG)